cmd/plugin/builder/inventory.go

```diff
@@ -47,7 +46,6 @@ func newInventoryInitCmd() *cobra.Command {
 				Repository:        piiFlags.Repository,
 				InventoryImageTag: piiFlags.InventoryImageTag,
 				Override:          piiFlags.Override,
-				ImgpkgOptions:     imgpkg.NewImgpkgCLIWrapper(),
```

Comment:
I think this needs to be replaced with 
`ImageOperationsImpl: carvelhelpers.NewImageOperationsImpl(),`

Reply:
Thanks for catching this.